README.md

```diff
@@ -63,6 +63,9 @@ config.omniauth :openid_connect, {
 | uid_field                    | The field of the user info response to be used as a unique id                                                                                                 | no       | 'sub'                      | "sub", "preferred_username"                         |
 | extra_authorize_params       | A hash of extra fixed parameters that will be merged to the authorization request                                                                             | no       | Hash                       | {"tenant" => "common"}                              |
 | allow_authorize_params       | A list of allowed dynamic parameters that will be merged to the authorization request                                                                         | no       | Array                      | [:screen_name]                                      |
+| pkce                         | Enable [PKCE flow](https://oauth.net/2/pkce/)                                                                                                                 | no       | false                      | one of: true, false                                 |
+| pkce_verifier                | Specify a custom PKCE verifier code.                                                                                                                          | no       | Random 32 character string | Proc.new { SecureRandom.hex(64) }                   |
```

Comment:
I see the default is set below is actually a 128-byte string? 

```ruby
irb(main):001:0> SecureRandom.hex(64).length
=> 128
```

Reply:
Good spot, bug from the code I copied. I think the default hex method returns a 32 char string so that's probably where the confusion came in. I'll update it.